assets/css/attractions.css

```diff
@@ -0,0 +1,12 @@
+/*This is the attractions css*/
+
+.tileItem {
+  width: 250px;
+}
+
+.attractionsContent section {
```

Comment:
Kept the attractions page full width. The rest of the pages have a max-width defined.